lib/encryptor.rb

```diff
@@ -4,29 +4,13 @@
 
 class Encryptor < Cryptor
 
-  attr_reader :encrypt_key, :encrypt_offset, :encrypt_shift
+  attr_reader :encrypt_key, :encrypt_offset, :shift
```

Comment:
changed the `encrypt_shift` instance variable to `shift` to make this instance variable class agnostic.